src/Templates/src/Microsoft.Maui.Templates.csproj

```diff
@@ -16,10 +15,33 @@
     <NoWarn>$(NoWarn);NU5128</NoWarn>
   </PropertyGroup>
 
-
   <ItemGroup>
     <Content Include="templates\**\*" Exclude="templates\**\bin\**;templates\**\obj\**" />
     <Compile Remove="**\*" />
   </ItemGroup>
 
+  <!-- Eventually replaced by eng/Version.targets -->
+  <Target Name="SetVersions" />
+
+  <Target Name="_UpdateTemplateVersions" BeforeTargets="GenerateNuspec" DependsOnTargets="SetVersions">
```

Comment:
This will automatically update the templates to be the package version without the `+` part.